components/server/src/workspace/workspace-starter.ts

```diff
@@ -818,6 +829,15 @@ export class WorkspaceStarter {
                     useLatest: user.additionalData?.ideSettings?.useLatestVersion,
                 },
             };
+            if (ideConfig.ideSettings && ideConfig.ideSettings.trim() !== "") {
+                try {
+                    const ideSettings: IDESettings = JSON.parse(ideConfig.ideSettings);
+                    configuration.ideConfig!.desktopIdeAlias = ideSettings.defaultIde;
```

Comment:
@svenefftinge Should we remove desktopIdeAlias? It is not used anywhere on main.

And instead add a new property like `ide` to reflect that it is any IDE.